pages/learn/manage/billing.md

```diff
@@ -24,7 +46,7 @@ At the bottom of this page, you'll find more information on the available [plans
 
 ## Dynamic billing
 
-From August 15th, 2018, and onward, we will be tracking usage dynamically. For any usage that is outside the limits defined by your plan, you will be billed. Dynamic billing for devices and users outside of your plan will occur every month, even for those on annual plans.
+For any usage that is outside the usage included in your billing plan, we will track your usage, and you will be billed dynamically in arrears. Dynamic billing for devices and users outside of your plan will occur every month, even for those on annual plans.
```

Comment:
```suggestion
Any usage that is outside the limits/offerings of the current billing plan, will be tracked and billed dynamically in arrears. Dynamic billing for devices and users outside of the organization's plan will occur every month, even for those with annual billing plans.
```

Reply:
I purposefully removed the word limits on this one. So I'll find a word that isn't limits but perhaps is clearer.